Resources/Prototypes/_Scp/Maps/mappool.yml

```
@@ -1,6 +1,6 @@
 - type: gameMapPool
   id: FireStationMapPool
   maps:
-  - ScpComplex
-  - ScpComplexCB
-  - ScpComplexMini
+  - ScpComplexCompact
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)